rundnsmasq.sh

```diff
@@ -15,13 +14,9 @@ mkdir -p /shared/html/pxelinux.cfg
 # Copy files to shared mount
 cp /tftpboot/undionly.kpxe /tftpboot/ipxe.efi /tftpboot/snponly.efi /shared/tftpboot
 
-# Copy IPv4 or IPv6 config
-cp /etc/dnsmasq.conf.ipv$IPV /etc/dnsmasq.conf
+# Template and write dnsmasq.conf
+python -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ))' </etc/dnsmasq.conf.j2 >/etc/dnsmasq.conf
 
-# Use configured values
-sed -i -e s/IRONIC_URL_HOST/${IRONIC_URL_HOST}/g -e s/HTTP_PORT/${HTTP_PORT}/g \
-       -e s/DHCP_RANGE/${DHCP_RANGE}/g -e s/PROVISIONING_INTERFACE/${PROVISIONING_INTERFACE}/g \
-       /etc/dnsmasq.conf
 for iface in $( echo "$DNSMASQ_EXCEPT_INTERFACE" | tr ',' ' '); do
```

Comment:
Replace this logic as well?

Reply:
I wanted to ask about this first, we only specify one interface today in interface=, so I'm not sure what this would be used for?

e.g. we say:
```
interface=ens2
```

so what's the point of except-interface

Reply:
No idea, and I'm not even sure this code works.. Need to `git blame` I guess.